src/infiniteGrid.js

```diff
@@ -207,19 +207,20 @@ eg.module("infiniteGrid", ["jQuery", eg, window, document, "Outlayer"], function
 	 * InfiniteGrid is composed using Outlayer and supports recycle-dom.
 	 * DOM elements are fixed even contents are added infinitely.
 	 * @group egjs
-	 * @ko 그리드 레이아웃을 구성하는 UI 컴포넌트. InfiniteGrid는 Outlayer로 구성되어 있다. 하지만, 이 컴포넌트는 recycle-dom을 지원한다.
-	 * 컨텐츠를 계속 증가하면 할수록 일정한 DOM 개수를 유지할수 있다.
+	 * @ko InfiniteGrid는 무한 그리드 레이아웃 UI 컴포넌트이다. 이 컴포넌트는 크게 2가지 기능을 제공한다.
+	 첫째, 석공이 벽돌을 맞추며 벽을 쌓듯이, 수평방향으로 차곡 차곡 카드를 쌓는 레이아웃 배치를 제공한다.
```

Comment:
@jongmoon good. same width is default.
I think description is more detail. simple is more better.